tests/test_core.py

```diff
@@ -1914,13 +1851,13 @@ def test_biggerswitch(self):
 59899: 598995989959899
 Success!''')
 
-  @no_wasm_backend()
+  @no_wasm_backend("WebAssembly hasn't implemented computed gotos")
```

Comment:
The output error was "error: <unknown>:0:0: in function main i32 (): WebAssembly hasn't implemented computed gotos", and I took its word for it. asm2wasm passes that test, so you're right that's misleading in the context of emscripten.